packages/components/organisms/f-checkout/src/services/addressService.js

```diff
@@ -70,6 +72,32 @@ function getAddressClosestToPostcode (postcode, addresses) {
     return address;
 }
 
+const isAddressInLocalStorage = () => {
+    if (window.localStorage) {
+        const address = window.localStorage.getItem(storedLocationKey) ? JSON.parse(window.localStorage.getItem(storedLocationKey)) : null;
+        return !!(address && (address.PostalCode || address.ZipCode));
+    }
+
+    return false;
+};
+
+function mapLocalStorageAddress (address) {
```

Comment:
Can this be renamed to `getMappedLocalStorageAddress`? Since it's returning something.